docs/tutorials/gallery_rest_api.py

```diff
@@ -115,7 +115,9 @@
 #    system.
 # 2. Perform the actual upload with the information from step 1.
 
-response = client.post("/document", json={"name": document_name}).raise_for_status()
+response = client.post(
+    "/document", json={"name": document_name}
+).raise_for_status()
```

Comment:
Without this formatting, we have the following in our current live docs:

![image](https://github.com/Quansight/ragna/assets/6849766/005bf717-4d17-4cee-8867-c6371038e7f7)